client-app/shared/account/components/order-status.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <VcChip :variant="orderStatus?.variant" :color="orderStatus?.color || defaultColor" truncate rounded>
-    {{ status }}
+    {{ displayValue || status }}
```

Comment:
It's better use `??`

Reply:
if displayValue is empty string it is better not to show it